assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -629,6 +629,9 @@ che.cors.allow_credentials=false
 che.factory.default_editor=eclipse/che-theia/next
 che.factory.default_plugins=eclipse/che-machine-exec-plugin/nightly
 
+# Devfile filenames to look on repository-based factories (like GitHub etc)
```

Comment:
I think it's very important to mention that the list is sorted by priority and 1st has the highest priority.